app/models/concerns/hyrax/ability/admin_set_ability.rb

```diff
@@ -2,28 +2,52 @@
 module Hyrax
   module Ability
     module AdminSetAbility
-      def admin_set_abilities # rubocop:disable Metrics/MethodLength
+      def admin_set_abilities # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
         if admin?
-          can :manage, AdminSet
+          can :manage, [AdminSet, Hyrax::AdministrativeSet]
           can :manage_any, AdminSet
+          can :manage_any, Hyrax::AdministrativeSet
           can :create_any, AdminSet
+          can :create_any, Hyrax::AdministrativeSet
           can :view_admin_show_any, AdminSet
+          can :view_admin_show_any, Hyrax::AdministrativeSet
         else
-          can :manage_any, AdminSet if Hyrax::Collections::PermissionsService.can_manage_any_admin_set?(ability: self)
-          can [:create_any, :create], AdminSet if Hyrax::CollectionTypes::PermissionsService.can_create_admin_set_collection_type?(ability: self)
-          can :view_admin_show_any, AdminSet if Hyrax::Collections::PermissionsService.can_view_admin_show_for_any_admin_set?(ability: self)
-
-          can [:edit, :update, :destroy], AdminSet do |admin_set| # for test by solr_doc, see solr_document_ability.rb
+          if Hyrax::Collections::PermissionsService.can_manage_any_admin_set?(ability: self)
+            can :manage_any, AdminSet
+            can :manage_any, Hyrax::AdministrativeSet
+          end
+          if Hyrax::CollectionTypes::PermissionsService.can_create_admin_set_collection_type?(ability: self)
+            can :create, [AdminSet, Hyrax::AdministrativeSet]
+            can :create_any, AdminSet
+            can :create_any, Hyrax::AdministrativeSet
+          end
+          if Hyrax::Collections::PermissionsService.can_view_admin_show_for_any_admin_set?(ability: self)
+            can :view_admin_show_any, AdminSet
+            can :view_admin_show_any, Hyrax::AdministrativeSet
+          end
+          # [:edit, :update, :destroy] for AdminSet is controlled by Hydra::Ability #edit_permissions
```

Comment:
The check for [:edit, :update, :destroy] for AF AdminSet was removed because the check already happens in Hydra::Ability.